content/case_histories/mt_isa/properties.rst

```diff
@@ -30,21 +30,21 @@ Overall we expect any mineralization unit in this area to be **conductive and ch
-+---------------------------+-------------------+------------------------------------------+--------------------------+
-| Moondarra Siltstone       |  Moderate         | Moderate  (~10)                          |  Low                     |
-+---------------------------+-------------------+------------------------------------------+--------------------------+
-| Breakaway Shale           |  High             | Low (~1)                                 |  Low-None                |
-+---------------------------+-------------------+------------------------------------------+--------------------------+
-| Mt Novit Horizon          |  High             | Low (~1)                                 |  High                    |
-+---------------------------+-------------------+------------------------------------------+--------------------------+
-| Surprise Creek Formation  |  Low              | High (~1000)                             |  None                    |
-+---------------------------+-------------------+------------------------------------------+--------------------------+
-| Eastern Creek Volcanics   |  Low              | High (~1000)                             |  High                    |
-+---------------------------+-------------------+------------------------------------------+--------------------------+
++---------------------------+-------------------+------------------------------------------+-------------------+
+|       **Rock Unit**       | **Conductivity**  | **Resistivity** (:math:`\Omega \cdot m`) | **Chargeability** |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Native Bee Siltstone      |  Moderate         | Moderate  (~10)                          |  Low              |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Moondarra Siltstone       |  Moderate         | Moderate  (~10)                          |  Low              |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Breakaway Shale           |  High             | Low (~1)                                 |  Low-None         |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Mt Novit Horizon          |  High             | Low (~1)                                 |  High             |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Surprise Creek Formation  |  Low              | High (~1000)                             |  None             |
++---------------------------+-------------------+------------------------------------------+-------------------+
+| Eastern Creek Volcanics   |  Low              | High (~1000)                             |  Low              |
```

Comment:
@fourndo , @dougoldenburg, @krisdavis : this is the update to confirm. Eastern Creek Volcanics == Low Chargeability??